tests/tests_geomstats/test_quotient_metric.py

```diff
@@ -11,154 +11,163 @@
 from geomstats.geometry.special_orthogonal import SpecialOrthogonal
 
 
+class BuresWassersteinBundle(GeneralLinear, FiberBundle):
```

Comment:
Should this go into the module `geometry`?

Reply:
Yann implemented the closed-form formulas for the BW metric, that's why I didn't think relevant to add it, but it could be in a notebook to exemplify the construction of a quotient metric

Reply:
Yessss 🎉 !